core/tests/blockstore/keyed-crypto.test.ts

```
@@ -6,11 +6,11 @@ import * as cborg from "cborg";
 import type { KeyBagProviderIndexedDB } from "@fireproof/core-gateways-indexeddb";
 import { mockLoader, MockSuperThis, mockSuperThis } from "../helpers.js";
 import { ensureSuperThis, keyedCryptoFactory, storeType2DataMetaWal } from "@fireproof/core-runtime";
-import { V2KeysItem, PARAM, StoreType } from "@fireproof/core-types-base";
+import { V2KeysItem, PARAM, StoreType, KeyBagIf, KeyWithFingerPrint, V2StorageKeyItem } from "@fireproof/core-types-base";
```

ESLint

[error] 'V2KeysItem' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)